### PR TITLE
fix: fix build definition of DatasetFieldUrn

### PR DESCRIPTION
**Pull request by jplaisted**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: li-utils/src/main/javaPegasus/com/linkedin/common/urn/DatasetFieldUrn.java [hunk 1]
```diff
@@ -55,22 +51,47 @@ public DatasetFieldUrn coerceOutput(Object object) throws TemplateOutputCastExce
...
+    this(new DatasetUrn(new DataPlatformUrn(dataPlatform), datasetName, fabricType), fieldPath);
+  }
+
   /**
-   * Creates a new instance of a {@link DatasetFieldUrn }.
+   * Creates a new instance of a {@link DatasetFieldUrn}.
    *
-   * @param dataset Dataset that this dataset field belongs to.
-   * @param fieldPath Dataset field path or column name
+   * @param datasetUrn dataset that this dataset field belongs to
```
Comment (jywadhwani) [inline li-utils/src/main/javaPegasus/com/linkedin/common/urn/DatasetFieldUrn.java:61]: nit: param is `dataset`

Comment (jywadhwani) [reply li-utils/src/main/javaPegasus/com/linkedin/common/urn/DatasetFieldUrn.java:61]: nvm, you changed input variable name too

Comment (jplaisted) [reply li-utils/src/main/javaPegasus/com/linkedin/common/urn/DatasetFieldUrn.java:61]: I did rename the param, but yeah, lets be consistent with the PDL. Thanks for pointing that out.

**Review by jywadhwani — COMMENTED** [review]
You don't need changes in `DatasetFieldUrnTest.java` ?

**Comment by jplaisted** [comment]
> You don't need changes in `DatasetFieldUrnTest.java` ?

They still pass, no

**Review by jywadhwani — APPROVED** [review]

**Pull request merged**